Documentation/WritingDocForExtension/Introduction.rst

```diff
@@ -18,6 +18,11 @@ extension contains a directory `Documentation` with at least the following files
 * `Includes.txt` : some basic definitions for sphinx
 * `Settings.cfg` : settings
 
+In addition, the `composer.json` of your extension should require the relevant version(s) of `typo3/cms-core`.
+
+Finally, to trigger documentation rendering you should add `https://docs-hook.typo3.org/` as a webhook to your
```

Comment:
> to trigger documentation rendering

I would add _"on the documentation server"_ to make it a little clearer. I will push another commit.